packages/fluentui/react-northstar/src/components/Datepicker/Datepicker.tsx

```diff
@@ -90,6 +96,9 @@ export interface DatepickerProps extends UIComponentProps, Partial<ICalendarStri
 
   /** Controls the calendar's 'selectedDate'. */
   selectedDate?: Date;
+
+  /** Marks which type of datepicker should be rendered. */
+  type?: DatepickerType;
```

Comment:
```suggestion
  appearance?: DatepickerAppearence;
```

For me type would be more of "DayPicker", "WeekPicker", "MonthPicker", etc.
@layershifter  @mnajdova WDYT?

Reply:
Or maybe we could follow `Button` and have 2 separate props `iconOnly` and `inputOnly` (similarly to `Button`'s `iconOnly` and `text` props)

Reply:
Changed.